api/capi/src/nnstreamer-capi-util.c

```diff
@@ -944,6 +945,11 @@ _ml_validate_model_file (char **model, unsigned int num_models)
     return ML_ERROR_INVALID_PARAMETER;
   }
 
+  if (g_file_test (model[0], G_FILE_TEST_IS_DIR)) {
```

Comment:
does this mean that the `model` for nnfw should be provided as `model folder path` and not `model file path`, then unittests also needs updating.

Reply:
or atleast a new unittest needed with input as model path.

Reply:
Both model file path and directory.
I added testcases to open nnfw single-shot with directory.